src/content/docs/develop/resources.mdx

```diff
@@ -94,14 +270,17 @@ tauri::Builder::default()
 fn hello(handle: tauri::AppHandle) -> String {
     let resource_path = handle.path().resolve("lang/de.json", BaseDirectory::Resource)?;
 
-    let file = std::fs::File::open(&resource_path).unwrap();
-    let lang_de: serde_json::Value = serde_json::from_reader(file).unwrap();
+    let json = std::fs::read_to_string(&resource_path).unwrap();
+    // Or when dealing with Android, use the file system plugin instead
+    // let json = handle.fs().read_to_string(&resource_path);
+
+    let lang_de: serde_json::Value = serde_json::from_str(json).unwrap();
 
     lang_de.get("hello").unwrap()
 }
 ```
 
-## Accessing files in JavaScript
+### JavaScript
 
 For the JavaScript side, you can either use a command like the one above and call it through `await invoke('hello')` or access the files using the [`plugin-fs`]
```

Comment:
```suggestion
For the JavaScript side, you can either use a command like the one above and call it through `await invoke('hello')` or access the files using the [`fs` plugin].
```
not sure if this suggestion works or if i missed some that don't show up in the diff.